sylt-parser/src/parser.rs

```diff
@@ -6,13 +6,22 @@ use std::path::{Path, PathBuf};
 use sylt_common::error::Error;
 use sylt_common::rc::Rc;
 use sylt_common::Type as RuntimeType;
-use sylt_tokenizer::{file_to_tokens, Token};
+use sylt_tokenizer::{PlacedToken, Token, file_to_tokens};
 
 pub mod expression;
 pub mod statement;
 pub use self::expression::{Expression, ExpressionKind};
 pub use self::statement::{Statement, StatementKind};
 
+pub use sylt_tokenizer::Span;
+
+static INVALID_TOKEN: Token = Token::EOF;
```

Comment:
This is a bit confusing. `EOF` isn't invalid. I don't think these constants add anything for readability.

Reply:
The EOF can be inlined, but the other static (the zero-span) needs to be a static. Otherwise we can't return a reference to it in `peek`: https://github.com/FredTheDino/sylt-lang/blob/fecb55230a1d60d8fc7e364aafd9da6d8ba2e4cb/sylt-parser/src/parser.rs#L254